<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavLabel</class>
 <widget class="QWidget" name="frmNavLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NavLabel" name="navLabel51">
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel52">
         <property name="text">
          <string>学生管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel53">
         <property name="text">
          <string>成绩查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="NavLabel" name="navLabel11">
         <property name="text">
          <string>当前温度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="NavLabel" name="navLabel21">
         <property name="text">
          <string>温度告警值</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="NavLabel" name="navLabel12">
         <property name="text">
          <string>当前湿度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="NavLabel" name="navLabel22">
         <property name="text">
          <string>湿度告警值</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="NavLabel" name="navLabel13">
         <property name="text">
          <string>当前压力</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="NavLabel" name="navLabel23">
         <property name="text">
          <string>压力告警值</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NavLabel" name="navLabel31">
         <property name="text">
          <string>当
前
温
度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel32">
         <property name="text">
          <string>当
前
湿
度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel33">
         <property name="text">
          <string>当
前
压
力</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel41">
         <property name="text">
          <string>当
前
温
度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel42">
         <property name="text">
          <string>当
前
湿
度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="NavLabel" name="navLabel43">
         <property name="text">
          <string>当
前
压
力</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavLabel</class>
   <extends>QLabel</extends>
   <header>navlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
